api/opencl_platform_layer.asciidoc

```diff
@@ -1423,13 +1423,22 @@ include::{generated}/api/version-notes/CL_DEVICE_GENERIC_ADDRESS_SPACE_SUPPORT.a
 // This query didn't exist pre-OpenCL 3.0, but:
 //Support for the generic address space is required for an OpenCL 2.0, 2.1, or 2.2 device.
 
-| {CL_DEVICE_DEVICE_ENQUEUE_SUPPORT_anchor}
+| {CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES_anchor}
 
-include::{generated}/api/version-notes/CL_DEVICE_DEVICE_ENQUEUE_SUPPORT.asciidoc[]
-  | cl_bool
-      | Is {CL_TRUE} if the device supports device-side enqueue and on-device queues, and {CL_FALSE} otherwise.
+include::{generated}/api/version-notes/CL_DEVICE_DEVICE_ENQUEUE_CAPABILITIES.asciidoc[]
+  | cl_device_device_enqueue_capabilities
+      | Describes device-side enqueue capabilities of the device.
+        This is a bit-field that describes one or more of the following
+        values:
+
+        {CL_DEVICE_QUEUE_SUPPORTED_anchor} - Device supports device-side enqueue and On-Device queues
+
+        {CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT_anchor} - Device supports a replaceable default On-Device queue
+
+        If {CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT} is set, {CL_DEVICE_QUEUE_SUPPORTED} should also be set.
```

Comment:
I think this should be stronger, and say that CL_DEVICE_QUEUE_SUPPORTED **must** also be set when CL_DEVICE_QUEUE_REPLACEABLE_DEFAULT is set, not merely that it **should** be set.

Reply:
Done